chain/messagepool/messagepool.go

```diff
@@ -667,7 +668,7 @@ func (mp *MessagePool) verifyMsgBeforeAdd(ctx context.Context, m *types.SignedMe
 	return publish, nil
 }
 
-func (mp *MessagePool) Push(ctx context.Context, m *types.SignedMessage) (cid.Cid, error) {
+func (mp *MessagePool) Push(ctx context.Context, m *types.SignedMessage, publish bool) (cid.Cid, error) {
```

Comment:
I don't see a similar change for `PushUntrusted`. Is that intentional?

Reply:
Also, could you leave a clear comment on what exactly this bool does?

Reply:
We shouldn't need it for PushUntrusted since its not used in syncing the message pool. I think this API should really be folded back into Push itself and add a untrusted param to it

Reply:
Yes I definitely agree :)